src/formats/notion/convert-to-md.ts

```diff
@@ -341,15 +392,46 @@ function replaceNestedTags(body: HTMLElement, tag: 'strong' | 'em') {
 	}
 }
 
-function splitBrsInFormatting(htmlString: string, tag: 'strong' | 'em') {
-	const tags = htmlString.match(new RegExp(`<${tag}>(.|\n)*</${tag}>`));
+/**
+ * Merges tags if identical tags are placed next to each other.
+ */
+function mergeAdjacentTags(body: HTMLElement, tagName: FormatTagName) {
+	const tags = body.findAll(tagName);
+	if (!tags) return;
+	const regex = new RegExp(`</${tagName}>( *)<${tagName}>`, 'g');
+	for (const tag of tags) {
+		if (!tag || !tag.parentElement) continue;
+		const parent = tag.parentElement;
+		let parentHTML = parent?.innerHTML;
+		parent.innerHTML = parentHTML?.replace(regex, '$1');
+	}
+}
+
+/**
+ * Strips leading <br> artificats created by Notion
+ * These often occur before strong | em | mark | del tags
+ */
+function stripLeadingBr(body: HTMLElement, tagName: FormatTagName) {
+	const tags = body.findAll(tagName);
+	if (!tags) return;
+	for (const tag of tags) {
+		const prevNode = tag.previousSibling;
+		prevNode?.nodeName == 'BR' && prevNode?.remove();
+	}
+}
+
+function splitBrsInFormatting(body: HTMLElement, tagName: FormatTagName) {
+	// Simpler to just use the HTML string for this replacement
+	let htmlString = body.innerHTML;
+	const tags = htmlString.match(new RegExp(`<${tagName}>.*?</${tagName}>`, 'sg'));
 	if (!tags) return;
-	for (let tag of tags.filter((tag) => tag.contains('<br />'))) {
+	for (let tag of tags.filter((tag) => tag.includes('<br>'))) {
```

Comment:
Notion has not changed the export format for `<br />`.
However, `convert-to-md.ts` does not interact directly with the original HTML, but the DOM object:
```typescript
const dom = parseHTML(text);
```
where 
```typescript
parseHTML(text) = new DOMParser().parseFromString(text, 'text/html').documentElement;
```
The DOM object is an abstract representation of the original HTML, so the way DOM tags are formatted in string form will depend on standard set by DOM implementation.
In this case (for HTML documents), the string form of `BR` tags is always formatted as `<br>`.
Even if we explicitly set `someElement.innerHTML = '<br />'`, the DOM will still parse this as `<br>`.

This is why the original function did not work, because the `body.innerHTML` string is formatted using the `<br>` standard as opposed to the formatting used in the original Notion file.

Reply:
Got it, thank you!